stdlib/argparse.pyi

```diff
@@ -668,6 +695,32 @@ class _SubParsersAction(Action, Generic[_ArgumentParserT]):
 
     # Note: `add_parser` accepts all kwargs of `ArgumentParser.__init__`. It also
     # accepts its own `help` and `aliases` kwargs.
+    if sys.version_info >= (3, 14):
+        def add_parser(
+            self,
+            name: str,
+            *,
+            deprecated: bool = False,
+            help: str | None = ...,
+            aliases: Sequence[str] = ...,
+            # Kwargs from ArgumentParser constructor
+            prog: str | None = ...,
+            usage: str | None = ...,
+            description: str | None = ...,
+            epilog: str | None = ...,
+            parents: Sequence[_ArgumentParserT] = ...,
+            formatter_class: _FormatterClass = ...,
+            prefix_chars: str = ...,
+            fromfile_prefix_chars: str | None = ...,
+            argument_default: Any = ...,
+            conflict_handler: str = ...,
+            add_help: bool = ...,
+            allow_abbrev: bool = ...,
+            exit_on_error: bool = ...,
+            suggest_on_error: bool = ...,
+            color: bool = ...,
```

Comment:
```suggestion
            suggest_on_error: bool = False,
            color: bool = False,
```
I think we didn't fill in defaults here because at runtime they go through `**kwargs`. Let's add defaults for the new parameters at least, and if you like we can add defaults for the rest at some point too.

Reply:
Would you prefer if I add the missing defaults in this PR or in a separate one?

Reply:
Let's do it in another PR, I'll merge this one